shakedown/dcos/package.py

```diff
@@ -110,7 +110,7 @@ def install_package(
         if pre_install_notes:
             print(pre_install_notes)
 
-        package_manager.install_app(pkg, options, service_name)
+        package_manager.install_app(pkg, options)
```

Comment:
If the intent is to keep the API for shakedown unchanged, I would recommend injecting the service name into the `options` dict here. Otherwise one has an inconsistency in the way in which the service name is specified.

There are a number of other places in this function where a mismatch between a service name specified in `options` and `service_name` as a parameter will cause unintended behaviour.